### DISCUSSION TITLE
how install karavan builder on podman ?

### DISCUSSION THREAD

**Discussion by nturri1306** [apache/camel-karavan#1429] [Q&A]
### Ask a question

how install builder on podman ? I tried but it stops

### Variant

None

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

None

**Comment by mgubaidullin** [comment]
Karavan does not support to work on Podman. 
Theoretically it might work if configure Podman socket and mount it accordingly.

More information https://github.com/containers/podman/blob/main/docs/tutorials/socket_activation.md#socket-activation-of-the-api-service
